src/App.js

```diff
@@ -1,26 +1,37 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import React from 'react';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+import './App.css';
 
-function App() {
+const Home = () => <div>Home</div>;
+const About = () => <div>About</div>;
+
+const MainMenu = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Link to="/">
+        <button type="button">home</button>
+      </Link>
+      <Link to="/about">
+        <button type="button">About</button>
+      </Link>
     </div>
   );
-}
+};
+
+const App = () => {
```

Comment:
we will need to be using "history", it is part of the react-router configuration.

https://stackoverflow.com/questions/42672842/how-to-get-history-on-react-router-v4